lib/entity_extractor_client.rb

```diff
@@ -0,0 +1,39 @@
+require 'rest-client'
+require 'json'
+
+class EntityExtractorClient
+  attr_reader :service_base_url
+
+  DEFAULT_OPEN_TIMEOUT_IN_SECONDS = 1
+  DEFAULT_READ_TIMEOUT_IN_SECONDS = 1
+
+  def initialize(service_base_url, options = {})
+    @service_base_url = URI.parse(service_base_url)
+    @options = default_options.merge(options)
+    @logger = options[:logger] || Logging.logger[self]
+  end
+
+  def call(document)
+    response = RestClient.post(extract_url, document, @options)
+    JSON.parse(response)
+  rescue RestClient::Exception => e
```

Comment:
I don't think we should be swallowing this exception.  We discussed maybe swallowing connection error in development environments (to allow developers to run migrations if the entity extractor isn't running).